lib/administrate/field/time.rb

```diff
@@ -0,0 +1,11 @@
+require_relative "base"
+
+module Administrate
+  module Field
+    class Time < Base
+      def default
+        data && data.strftime("%H:%M:%S")
```

Comment:
@nickcharlton unfortunately I wasn't able to find any. I checked out here:
https://github.com/rails/rails/blob/master/activesupport/lib/active_support/locale/en.yml